grr-ui/src/templates/AssetMgmt/AssetMgmt.js

```diff
@@ -23,6 +23,10 @@ class AssetMgmt extends Component {
 
     }
 
+    selectBackground (backgroundTitle) {
+        this.setState({selectedBackground: backgroundTitle});
```

Comment:
Could the name "selectedBackground" be changed to "currentBackground", "background", "backgroundImage", or another name. selectedbackground was confusing to me when there is also a selectBackground method.

Don't know if this is too nit-picky.

Reply:
Definitely understandable. Fixed here: 7b8e920